source/yats/include/yats/AbstractConnector.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+namespace yats
+{
+class AbstractTaskConfigurator;
+
+class AbstractConnector
```

Comment:
Remove this class and merge its implementation into the other AbtractConnectors.

Reply:
The AbstractConnector class is required because both, AbstractInputConnector and AbstractOutputConnector, have the member m_owner. Furthermore AbstractInputConnector and AbstractOutputConnector are both connectors and should have the same base class.